common/templates/common/sample_djangre.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base_templates/base.html' %}
```

Comment:
🎨 The examples and the feature should share a name.  If the feature is going the referred to as "djagno_react", the view should be "sample_django_react".  Alternatively, call the feature "djangre"